pkg/kn/commands/service/configuration_edit_flags.go

```diff
@@ -480,3 +486,32 @@ func (p *ConfigurationEditFlags) AnyMutation(cmd *cobra.Command) bool {
 	}
 	return false
 }
+
+func (p *ConfigurationEditFlags) ScaleConversion(scale string) (scaleMin int, scaleMax int, err error) {
```

Comment:
Do you want the method `ScaleConversion` to be visible outside package? If the method is not used by any other packages, I suggest to change the method name to start with a lower case letter, e.g. `scaleConversion`